### PR TITLE
[DOCS] migrate ILM to DSL headings and TLDR

### PR DESCRIPTION
**Pull request by andreidan**
This adds some headings and a TL;DR section to the
migration to DSL tutorial.

### CODE REVIEW

File: docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc [hunk 1]
```diff
@@ -153,10 +169,13 @@ currently richer in features).
...
 in the index template).
 
+[discrete]
+[[migrate-from-ilm-to-dsl]]
+==== Migrate data stream to data stream lifecycle
 To migrate the `dsl-data-stream` to data stream lifecycle we'll have to execute
 two steps:
 
-1. Update the index template that's backing the index template to configure <<index-lifecycle-prefer-ilm, prefer_ilm>> 
+1. Update the index template that's backing the data stream to configure <<index-lifecycle-prefer-ilm, prefer_ilm>>
```
Comment (tylerperk) [inline docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc:178]:
```suggestion
1. Update the index template that's backing the data stream to set <<index-lifecycle-prefer-ilm, prefer_ilm>> 
```

**Review by tylerperk — CHANGES_REQUESTED** [review]
Thanks for adding the summary and headers. Please also make "data stream lifecycle" always lowercase as discussed, and "Index Lifecycle Management" should always be capitalized.

File: docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc [hunk 2]
```diff
@@ -13,6 +13,22 @@ being managed by Data stream lifecycle. As we'll see, {ilm-cap} and Data stream
...
 
+[discrete]
+[[migrate-dsl-ilm-tldr]]
+==== TL;DR
+To migrate a data stream from {ilm-cap} to  data stream lifecycle we'll have to execute
+two steps:
+
+1. Update the index template that's backing the data stream to configure <<index-lifecycle-prefer-ilm, prefer_ilm>> 
+to `false`, and to configure data stream lifecycle. 
+2. Configure the data stream lifecycle for the _existing_ data streams using
```
[the remark this answers is not in the document]
Comment (andreidan) [reply docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc:24]: They're likely more data streams in the system. Why do you think we should be singular here?

**Comment by andreidan** [comment]
@elasticmachine update branch

File: docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
+being managed by data stream lifecycle. As we'll see, {ilm-init} and data stream lifecycle
+can co-manage a data stream; however, an index can only be managed by one system at 
 a time.
 
-Let's first create a data stream with two backing indices managed by {ilm-cap}.
-We first create an {ilm-cap} policy:
+[discrete]
+[[migrate-dsl-ilm-tldr]]
+==== TL;DR
+To migrate a data stream from {ilm-init} to  data stream lifecycle we'll have to execute
```
Comment (gmarouli) [inline docs/reference/data-streams/lifecycle/tutorial-migrate-data-stream-from-ilm-to-dsl.asciidoc:19]:
```suggestion
To migrate a data stream from {ilm-init} to data stream lifecycle we'll have to execute
```

**Review by gmarouli — APPROVED** [review]
LGTM 🚀 , I love a good TLDR :) Thank you @andreidan

**Comment by andreidan** [comment]
>  I love a good TLDR

This was @tylerperk 's idea  🚀

**Pull request merged**